app/javascript/controllers/event_send_message_form_controller.js

```diff
@@ -0,0 +1,36 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+    static targets = ["scheduledAtForm", "radioButtonSendNowFalse", "radioButtonSendNowTrue", 'checkBoxAutoDone', "dateFieldScheduletAt"]
+    connect() {
+        this.scheduledAtFormTarget.classList.add("hidden");
+        this.checkBoxAutoDoneTarget.checked = true
+    }
+
+    toggleRadioButton(event) {
+        this.resetCheckRadioDiv() 
+        event.currentTarget.ariaChecked = 'true'
+        var doneInput = event.currentTarget.querySelector('input[type="radio"]');
+        var done = doneInput.value === 'true';
+        this.radioButtonSendNowTrueTarget.checked = done;
+        this.radioButtonSendNowFalseTarget.checked = !done;
+        if (done) {
+            // this.scheduledAtFormTarget.style.display = 'none';
+            this.scheduledAtFormTarget.classList.add("hidden");
+            this.checkBoxAutoDoneTarget.checked = true
+            this.dateFieldScheduletAtTarget.value = ''
+            
+        } else {
+            // this.scheduledAtFormTarget.style.display = '';
+            this.scheduledAtFormTarget.classList.remove("hidden");
+            this.checkBoxAutoDoneTarget.checked = false
+        }
+
+    }
+    resetCheckRadioDiv() {
+        this.element.querySelectorAll('.radio-button-div').forEach((x) => {
+            x.setAttribute('aria-checked', 'false');
+        });
```

Comment:
Aqui vai ser sempre um né?
Acho que não precisaria fazer esse forEach, talvez como:
```suggestion
        this.element.querySelector('.radio-button-div').setAttribute('aria-checked', 'false');
```

Reply:
Temos que conversar sobre isso

Reply:
Esse codigo é para tirar o estilo de selecionado das divs que contem o radio button:
![image](https://github.com/douglara/woofed-crm/assets/45243355/bfd5e801-bd2e-4935-8df5-28d0d61b02c5)
Esse estilo roxo é aplicado quando uma div tem o aria-checked = true. Eu reseto de todos os dois e seto na linha seguinte a div que será selecionada de acordo com a div que o usuário clicou. Fiz algumas alterações no nome para ficar mais claro, esse caso eu renomeei para `send-now-wrapper`./